Devices/EMX/Deployment.cpp

```diff
@@ -19,7 +19,7 @@
 #define TOTAL_DEPLOYMENT_CONTROLLERS 1
 
 const char* deploymentApiNames[TOTAL_DEPLOYMENT_CONTROLLERS] = {
-    "GHIElectronics.TinyCLR.NativeApis.LPC24.StorageController\\0"
+    "GHIElectronics.TinyCLR.NativeApis.AT49BV322DT.ExternalFlash.StorageController\\0"
```

Comment:
Should be `"GHIElectronics.TinyCLR.NativeApis.AT49BV322DT.StorageController\\0"`